cmd/gatekeeper-test-alpha/gatekeeper-test-alpha.go

```diff
@@ -49,19 +67,25 @@ var rootCmd = &cobra.Command{
 		}
 
 		isFailure := false
-		for _, f := range testFiles {
-			result := gktest.Run(f, filter)
-			// If Result contains an error status, it is safe to execute tests in other
-			// files so we can continue execution.
-			isFailure = isFailure || result.IsFailure()
-			fmt.Println(result.String())
+		for _, s := range suites {
+			if !filter.Suite(s) {
```

Comment:
This reads a bit oddly ("if not filter suite, ignore"), how can we self-document that `filter.Suite(suite)`? Reverse the polarity? Change the `filter` name to `selector`?

Reply:
Done.